src/Policies/RatePolicy.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Bookings\Policies;
+
+use Illuminate\Auth\Access\HandlesAuthorization;
+use Tipoff\Bookings\Models\Rate;
+use Tipoff\Support\Contracts\Models\UserInterface;
+
+class RatePolicy
+{
+    use HandlesAuthorization;
+
+    public function viewAny(UserInterface $user): bool
+    {
+        return $user->hasPermissionTo('view rates') ? true : false;
+    }
+
+    public function view(UserInterface $user, Rate $rate): bool
+    {
+        return $user->hasPermissionTo('view rates') ? true : false;
+    }
+
+    public function create(UserInterface $user): bool
+    {
+        return false;
```

Comment:
We should also allow rates to be created.

They definitely shouldn't be able to be deleted. I think we should also not allow them to be updated unless we limit what fields can be updated on the rate in the Nova resource.

Reply:
I might take this comment back.

@sl0wik Do you plan on this package to not dictate things like tipoff/checkout and instead just receive the Rate from the EscapeRoomRate or other package that implements the tipoff/bookings service?

Reply:
The boundary between those packages is very fluid so always hard to say what should be where. In theory - most of stuff should  be in EscapeRoom. Putting it in booking would make it more universal, and putting to EscapeRoom more customizable. If its up to me I would kick significant amount of logic to EscapeRoom. Including things like status (it would be BookingStatus interface).

Reply:
I'm fine with that. I agree that the goal of Bookings should be more universal and that tipoff/scheduler and tipoff/escape-room should have the industry-specific stuff.

My preference would be to use generic terms in the bookings package and add an ER or EscapeRoom prefix to the other packages.

Reply:
(I will quickly draw whimsical)